lib/src/main/kotlin/io/bloco/faker/components/Internet.kt

```diff
@@ -0,0 +1,156 @@
+package io.bloco.faker.components
+
+import io.bloco.faker.FakerComponent
+import io.bloco.faker.FakerData
+import java.util.*
```

Comment:
avoid doing this imports :) I think theres a configuration in the IDE to avoid this